megamek/src/megamek/common/Terrain.java

```diff
@@ -386,29 +388,52 @@ public int movementCost(Entity e) {
                 } else {
                     mp = 1;
                 }
+
+                if (isCrossCountry) {
+                    if (((moveMode == EntityMovementMode.HOVER) || (moveMode == EntityMovementMode.TRACKED))
+                            && (level == 6)) {
+                        mp *= 2;
+                    } else if ((moveMode == EntityMovementMode.WHEELED)) {
+                        mp *= 2;
+                    }
+                }
+
                 if ((e instanceof Mech) && e.isSuperHeavy()) {
                     mp -= 1;
                 }
+
                 if (e.hasAbility(OptionsConstants.PILOT_TM_MOUNTAINEER)) {
                     mp -= 1;
                 }
+
                 if ((e.hasAbility(OptionsConstants.INFANTRY_FOOT_CAV)
                         && (moveMode == EntityMovementMode.INF_LEG))) {
                     mp -= 1;
                 }
                 return Math.max(0, mp);
             case Terrains.WOODS:
                 mp = level;
+                if (isCrossCountry) {
+                    if ((level == 1) && ((moveMode == EntityMovementMode.HOVER) || (moveMode == EntityMovementMode.WHEELED))) {
```

Comment:
Same as the other, OR-ing this looks even easier as level can't be 0

Reply:
But it can be greater than one, and certain vehicles can't travel is certain woods levels. Woods have 3 levels, 1 is light woods, 2 is heavy woods, and 3 is ultra heavy woods. Hover and Wheeled vehicles cannot enter woods at all while tracked vehicles can only enter light woods.

Reply:
Hmm I feel we're talking past each other here.
With crossCountry, what this checks is:
if Level 1 and h/w --> mp*2
-also- if Level > 1 --> mp*2
That is exactly the same as if (Level 1 and h/w) -or- Level > 1 --> mp*2
Which is the same as: if level > 1 -or- h/w --> mp*2 (assuming level must be > 0)
(I have not thought about the rules, only the code as it is)

Reply:
italic text not intentional